### PR TITLE
Support ONNX export for Squeeznet Model

### PR DESCRIPTION
**Pull request by lara-hdr**
https://github.com/pytorch/vision/issues/690

### CODE REVIEW

**Comment by fmassa** [comment]
Hi Lara,

As discussed in #690, this PR will change the results from the model. Additionally, the Dropout layer is necessary for training, and removing it on  the model definition is a pretty big breaking change (training new models won't work at all I believe).
